SPECS/cloud-hypervisor-cvm/update_src

```diff
@@ -0,0 +1,72 @@
+#/bin/bash
```

Comment:
is this file required?

Reply:
Double checked and no. Same as https://github.com/microsoft/azurelinux/pull/9161/files#discussion_r1621506207. We can get the source tar by looking at the tags page in https://github.com/microsoft/cloud-hypervisor/tags , and we can generate the vendor tar by following https://github.com/microsoft/azurelinux/pull/9261/files#diff-5d949bbc496656a772fe7a22601f9837efcc1dffcf513a7ccffd6aa6d53ae2e4R16-R22 (adding this comment to cvm spec)

May be worth documenting this comment in the form of a useful script for the future. It's 3 or 4 straightforward commands